packages/common/src/controllers/AccountController.ts

```diff
@@ -418,6 +418,12 @@ export default class AccountController {
 
     let pendingOffer: Offer | null = null;
 
+    if (!activeSubscription && !!retry && retry > 0) {
+      const retryDelay = 1500; // Any initial delay has already occured, so we can set this to a fixed value
```

Comment:
Should we do some sort of backoff on this? Like `retryDelay *= 2`, so we cast a wider net?

Reply:
Interesting suggestion. I'd have to keep some state. Perhaps a good option for an updated version as suggested?